Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -456,6 +457,28 @@ def users_to_entry(title, response, next_page_token=None):
     }
 
 
+def labels_to_entry(title, response, user_key):
```

Comment:
I think this function is redundant, you can do it in the ```list_labels_command```.

Reply:
I was just following the convention of the other code. There seems to be a nice formatting function for each call.

Reply:
Its not super important, if you think it would be better to still use that function, i am in.